utils.py

```diff
@@ -856,6 +856,55 @@ def compute_list_difference(list_a, list_b):
     return list(set(list_a) - set(list_b))
 
 
+def is_appengine_simulated_environment():
```

Comment:
is_local_server_environment or is_running_locally, perhaps

(When I first read "appengine simulated", I thought it meant the opposite to what you want it to mean. Probably because it has "appengine" in the name, which is kinda strongly associated with the cloud service.)

Reply:
I see. Fixed.